dismob/log.py

```
@@ -143,32 +148,25 @@ async def safe_send_message(channel: discord.TextChannel, content: str = None, e
     except discord.InteractionResponded:
         # Use followup if already responded
-        try:
-            return await get_rate_limiter().execute_request(
-                interaction.followup.send(content, embed=embed, ephemeral=ephemeral),
-                route='POST /webhooks/{application_id}/{interaction_token}',
-                major_params={'application_id': interaction.application_id}
-            )
-        except Exception as e:
-            error(f"Error in the followup: {e}")
+        safe_followup(interaction, content, embed, view, file, ephemeral)
     except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)